src/app/services/auth/auth.service.ts

```diff
@@ -41,6 +41,8 @@ export class AuthService {
   @LocalStorage() private token: string;
   protected loggedInUser$ = new BehaviorSubject<LoggedInUser>(null);
 
+  instantAuthTokenValue$ = new BehaviorSubject<string>(this.window.localStorage.getItem('ngx-webstorage|token'));
```

Comment:
This relies on internal implementation of a 3rd party. Is there a way to just read value from `token`?